components/Pages/Payruns/SinglePayRun/PayRunItem.js

```diff
@@ -101,11 +132,19 @@ export const PayRunItem = ({
         <HoldPaymentDialog
           invoiceId={invoiceId}
           payRunId={payRunId}
-          isOpen={invoiceId}
           update={updateData}
-          setIsOpened={() => setInvoiceId('')}
+          isOpen={openedModal === 'Hold'}
+          setIsOpened={closeModal}
         />
       )}
+      <ApproveDeclineModal
+        setOpenedModal={closeModal}
+        openedModal={openedModal !== 'Hold' && openedModal}
+        payRunId={payRunId}
+        title="Invoice"
+        request={rejectInvoiceStatus}
+        update={update}
```

Comment:
rename update prop to something more descriptive

Reply:
renamed update to isActivePayRun